modules/cli-manager-updating-plugin.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/cli_manager/cli-manager-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cli-manager-updating-plugin_{context}"]
+= Updating CLI plugins with the {cli-manager}
+
+You can update a plugin that was installed for the OpenShift CLI (`oc`) with the {cli-manager}.
+
+.Prerequisites
+
+* You have installed Krew by following the link:https://krew.sigs.k8s.io/docs/user-guide/setup/install/[installation procedure] in the Krew documentation.
+* You are running {product-title} 4.17 or later.
+* The CLI Manager is installed.
+* You've added the custom index to Krew.
```

Comment:
This procedure is done by a user to update its plugins to whatever newer version the admin update to, right?

And the custom index task would have been something that their admin did, not them, correct? So I don't think it's right to say that they added the custom index to Krew. That should have already been taken care of by their admin.